Japx/Classes/Objc/JapxObjc.swift

```diff
@@ -0,0 +1,131 @@
+//
+//  JapxWrapper.swift
+//  Japx
+//
+//  Created by Nikola Majcen on 29/01/2019.
+//
+
+import Foundation
+
+@objc public class JAPXDecoder: NSObject {
```

Comment:
In Objective-C max first four letters are uppercased. As Vlaho said, it would be better `JapxDecoder/Encoder`

Reply:
I've talked with @Truba about naming issue, and it is used `JAPXDecoder` because it is already defined `JapxDecoder` in the source code which creates issues when the library needs to be built.